nixos/doc/manual/release-notes/rl-2405.section.md

```diff
@@ -265,6 +265,8 @@ The pre-existing [services.ankisyncd](#opt-services.ankisyncd.enable) has been m
 
 - `hare` may now be cross-compiled. For that to work, however, `haredoc` needed to stop being built together with it. Thus, the latter is now its own package with the name of `haredoc`.
 
+- `services.redis.vmOverCommit` now defaults to `true` and no longer enforces Transparent Hugepages to be disabled as long as they are set to `madvise`.
```

Comment:
>  as long as they are set to `madvise`

I don't understand this part. This likely needs a link for more context.

Reply:
madvise is the kernel default and redis only works with that. If anything else is set the following error should be triggered

https://github.com/redis/redis/blob/f95031c4733078788063de775c968b6dc85522c0/src/syscheck.c#L160-L164

https://www.kernel.org/doc/html/v4.18/admin-guide/mm/transhuge.html#:~:text=the%20near%20future.-,madvise,-will%20enter%20direct

Reply:
I've reworded this section a bit.